composer.json

```diff
@@ -13,8 +13,8 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zend-config": "2.2.3",
-        "zendframework/zend-di": "2.2.3"
+        "zendframework/zend-config": "~2.3",
```

Comment:
@lcf this does do that (via the tilde constraint): https://getcomposer.org/doc/01-basic-usage.md